bots.go

```diff
@@ -35,19 +35,28 @@ func (api *Client) botRequest(ctx context.Context, path string, values url.Value
 	return response, nil
 }
 
+type GetBotInfoParameters struct {
+	Bot    string
+	TeamID string
+}
+
 // GetBotInfo will retrieve the complete bot information
-func (api *Client) GetBotInfo(bot string) (*Bot, error) {
-	return api.GetBotInfoContext(context.Background(), bot)
+func (api *Client) GetBotInfo(parameters GetBotInfoParameters) (*Bot, error) {
```

Comment:
The only thing I want to point out is that this is a breaking change in a library. I can't think of a better way to do this, so it's worth thinking about updating the version.

Reply:
I think it's fine to do a breaking change while the version is `v0.x.y`.
Once the library hits `v1` that'll be another thing.

Additionally, I'm thinking of overhauling the params as the inconsistent use or url params bugs be a bit (& that'll also be breaking, so I'd reserve a version bump for that).

If that's the only thing could this PR be merged & released?

Reply:
I don't have the merge rights, though